antismash/modules/pfam2go/test/test_pfam2go.py

```diff
@@ -115,22 +115,11 @@ def test_blank_records(self):
             assert not gos_for_no_ids
 
     def test_get_gos_id_handling(self):
-        pfams = {'PF00015.42': FeatureLocation(0, 3), 'PF00015_42': FeatureLocation(0, 3),
-                 'PF0015.42': FeatureLocation(0, 3), 'PPF00015.42': FeatureLocation(0, 3)}
+        pfams = {'PF00015.42': FeatureLocation(0, 3),
+                 'PF0015.42': FeatureLocation(0, 3)}
         fake_record = set_dummy_with_pfams(pfams)
-        # are wrong PFAM ids logged?
-        with self.assertLogs() as log_cm:
-            gos_for_fake_pfam = pfam2go.get_gos_for_pfams(fake_record)
-            assert 'Pfam id PF00015_42 is not a valid Pfam id, skipping' in str(log_cm.output) \
-                   and 'Pfam id PF0015 is not a valid Pfam id, skipping' in str(log_cm.output) \
-                   and 'Pfam id PPF00015 is not a valid Pfam id, skipping' in str(log_cm.output)
-        for all_ontologies in gos_for_fake_pfam.values():
-            for ontologies in all_ontologies:
-                # catches both stripping version number and broken ID not being included
-                assert ontologies.pfam.isalnum()
-                assert ontologies.pfam.startswith('PF')
-                for ontology in ontologies.go_entries:
-                    assert ontology.id in self.known_connections[ontologies.pfam]
+        with self.assertRaises(ValueError):
```

Comment:
This ought to be a `self.assertRaisesRegex(ValueError, 'Invalid PFam id')`, to check that it's the ValueError you expect and not some other ValueError, since they're relatively common.